packages/vue/src/types.ts

```diff
@@ -41,12 +41,6 @@ export interface VueOptions {
    */
   attachProps: boolean;
 
-  /**
-   * When set to `true`, original Vue's `logError` will be called as well.
-   * https://github.com/vuejs/vue/blob/c2b1cfe9ccd08835f2d99f6ce60f67b4de55187f/src/core/util/error.js#L38-L48
-   */
-  logErrors: boolean;
```

Comment:
I wonder if there is an easy way to deprecate this in v8

Reply:
I deprecated it in this PR: https://github.com/getsentry/sentry-javascript/pull/14943